algorithms/kernel/low_order_moments/low_order_moments_estimates_batch.i

```diff
@@ -247,71 +247,75 @@ Status compute_estimates(NumericTable * dataTable, Result * result)
     const size_t numRowsBlocks      = _cd.nVectors / numRowsInBlock;
     const size_t numRowsInLastBlock = numRowsInBlock + (_cd.nVectors - numRowsBlocks * numRowsInBlock);
 
+    DAAL_ITTNOTIFY_SCOPED_TASK(LowOrderMomentsBatchTask.compute);
```

Comment:
Do you really need this task?

Reply:
Yes, it aggregates all activities related to compute, and added here for aligning with GPU branch.

Reply:
Yes, it aggregates all activities related to compute, and added here for aligning with GPU branch.